src/reporting/clients/IExtractionClient.ts

```diff
@@ -59,4 +59,33 @@ export interface IExtractionClient {
     accessToken: AccessToken,
     jobId: string
   ): Promise<ExtractionStatus>;
+
+  /**
+   * Gets Extractions of an IModel. This method returns the full list of Extractions.
+   * @param {string} iModelId The iModel Id.
+   * @param {string} accessToken OAuth access token with scope `insights:read`.
+   * @param {number} top The number of entities to load per page.
+   * @memberof ReportingClient
+   * @link https://developer.bentley.com/apis/insights/operations/get-extraction-history/
+   */
+  getExtractionHistory(
```

Comment:
Bump patch version in the package.json file so that we can release this package.